Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -319,6 +319,13 @@ void MainWindow::InitControllers()
     return;
 
   g_controller_interface.Initialize(GetWindowSystemInfo(windowHandle()));
+  if (!g_controller_interface.HasDefaultDevice())
+  {
+    // Note that the CI default device could be still temporarily removed at any time
+    WARN_LOG(CONTROLLERINTERFACE,
+             "No default divice has been added in time. EmulatedController(s) defaulting"
+             " adds bindings made for a specific default device per platform");
```

Comment:
The second sentence here doesn't make sense to me.

Also, `device` is misspelled in the first sentence.